ale/rotation.py

```diff
@@ -77,7 +83,7 @@ def __mul__(self, other):
                 Another rotation object, it can be constant or time dependent.
         """
         if self.source != other.dest:
-            raise ValueError("Destination frame of first rotation is not the same as source frame of second rotation.")
+            raise ValueError("Destination frame of first rotation {} is not the same as source frame of second rotation {}.".format(other.dest, self.source))
         if isinstance(other, ConstantRotation):
             new_rot = self._rot * other._rot
```

Comment:
Why are these being treated as private?

Reply:
`_rot` is the actual implementation of the rotation as a scipy Rotation object. This provides us better encapsulation.